src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -3250,6 +3324,283 @@ namespace InternalHeatGains {
+				//Initialize local variables
+				IsAutosize = false;
+				SurfNum = 0;
+				LowTemperatureDes = 0.0;
+				LowTemperatureUser = 0.0;
+				HighTemperatureDes = 0.0;
+				HighTemperatureUser = 0.0;
+				CapatLowTemperatureDes = 0.0;
+				CapatLowTemperatureUser = 0.0;
+				CapatHighTemperatureDes = 0.0; 
+				CapatHighTemperatureUser = 0.0;
+				ZnInfilSensLoad = 0.0;
+				ZnVentSensLoad = 0.0;
+				NZ = ZoneBBHeat(Loop).ZonePtr;
+
+				if (ZoneBBHeat(Loop).LowTemperature == AutoSize) {
+					IsAutosize = true;
+				}
+
+				LowTemperatureDes = ZoneBBHeat(Loop).ZnMinOutTemp;
+				if (IsAutosize) {
+					ZoneBBHeat(Loop).LowTemperature = LowTemperatureDes;
+					ReportSizingOutput("ZoneBaseboard:OutdoorTemperatureControlled", ZoneBBHeat(Loop).Name, "Design Size Low Temperature [C]", LowTemperatureDes);
+				} else {
+					LowTemperatureUser = ZoneBBHeat(Loop).LowTemperature;
+					ReportSizingOutput("ZoneBaseboard:OutdoorTemperatureControlled", ZoneBBHeat(Loop).Name, "Design Size Low Temperature [C]", LowTemperatureDes, "User-Specified Low Temperature [C]", LowTemperatureUser);
+					if (DisplayExtraWarnings) {
+						if ((abs(LowTemperatureDes - LowTemperatureUser)/LowTemperatureUser) > AutoVsHardSizingDeltaTempThreshold) {
+							ShowMessage("SizeZoneBaseboard:OutdoorTemperatureControlled: Potential issue with equipment sizing for " + trim(ZoneBBHeat(Loop).Name));
+							ShowContinueError("User-Specified Low Temperature of " + RoundSigDigits(LowTemperatureUser,2) + " [C]");
+							ShowContinueError("differs from Design Size Low Temperature of " + RoundSigDigits(LowTemperatureDes,2) + " [C]");
+							ShowContinueError("This may, or may not, indicate mismatched component sizes.");
+							ShowContinueError("Verify that the value entered is intended and is consistent with other components.");
+						}
+					}
+				}
+
+
+				IsAutosize = false;
+				if (ZoneBBHeat(Loop).HighTemperature == AutoSize) {
+					IsAutosize = true;
+				}
+
+				HighTemperatureDes = ZoneBBHeat(Loop).ZnHtgSetTemp;
+				if (IsAutosize) {
+					ZoneBBHeat(Loop).HighTemperature = HighTemperatureDes;
+					ReportSizingOutput("ZoneBaseboard:OutdoorTemperatureControlled", ZoneBBHeat(Loop).Name, "Design Size High Temperature [C]", HighTemperatureDes);
+				} else {
+					HighTemperatureUser = ZoneBBHeat(Loop).HighTemperature;
+					ReportSizingOutput("ZoneBaseboard:OutdoorTemperatureControlled", ZoneBBHeat(Loop).Name, "Design Size High Temperature [C]", HighTemperatureDes, "User-Specified High Temperature [C]", HighTemperatureUser);
+					if (DisplayExtraWarnings) {
+					  if ((abs(HighTemperatureDes - HighTemperatureUser)/HighTemperatureUser) > AutoVsHardSizingDeltaTempThreshold) {
+						ShowMessage("SizeZoneBaseboard:OutdoorTemperatureControlled: Potential issue with equipment sizing for " + ZoneBBHeat(Loop).Name);
+						ShowContinueError("User-Specified High Temperature of " + RoundSigDigits(HighTemperatureUser,2) + " [C]");
```

Comment:
@Myoldmopar @larryscheier Here's the missing code for sizing the OA controlled baseboard. Ref #5344